contribs/parking/src/main/resources/parkingsearch/config.xml

```diff
@@ -31,7 +31,7 @@
 		<param name="flowCapacityFactor" value="1.0"/>
 		<param name="storageCapacityFactor" value="1.0"/>
 		<param name="stuckTime" value="10"/>
-		<param name="removeStuckVehicles" value="yes"/>
+		<param name="removeStuckVehicles" value="false"/>
```

Comment:
This is a different example: the aim was to have the removal enabled, but actually you cannot remove stuck vehicles in DVRP, so "yes" worked properly (i.e. as "false")